_posts/2017-11-29-cocreation-guidelines.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Co-creating in non-hierarchical teams"
+published: true
+excerpt: "A lot of our work is done by volunteers and their backgrounds vary."
+tags:
+- co-creation
+- popular education
+- community agreements
+- anarchism
+- volunteer
+
+---
+
+A lot of our work is done by volunteers who come from different backgrounds, and many of whom are not "technical" in the conventional sense. We believe that this work is only tangentially technical, and at it's core, this work is about building community and protecting one another. In our teams, we want to create space for everyone to show up with their own skills, [to teach others and to learn from others](https://en.wikipedia.org/wiki/Popular_education). We also want to avoid replicating the toxic, oppressive elements of tech culture, such as the side-lining of ["soft skills"](http://emilygorcenski.com/2016/03/15/Commodities-Culture-and-Code/) (among many other things).
```

Comment:
Suggestion:

A lot of our work is done by volunteers who come from different backgrounds, ~~and~~ many of whom are not "technical" in the conventional sense.